integration_tests/pages/refer/taskList.ts

```diff
@@ -50,4 +50,15 @@ export default class TaskListPage extends Page {
       })
     })
   }
+
+  shouldHaveConfirmedOasys() {
+    cy.get('.moj-task-list > li:nth-child(2) .moj-task-list__item:nth-child(2)').then(
```

Comment:
This feels like a prime candidate for using a test `id`

Reply:
(I wrote this while in review mode by mistake so my first message was pending - including it here for context!)

Yeah - I feel like I had a problem adding the tag to the right element here though. I'll check

---

Yeah the problem is the `attributes` field on tags adds it to the `strong` element not the `.moj-task-list__item` element:

```html
<li class="moj-task-list__item">
  <a class="moj-task-list__task-name" href="/referrals/85ed5f2b-d912-4b47-8db8-349a98042708/oasysConfirmed">Confirm the OASys information</a>
  <strong class="govuk-tag govuk-tag--grey moj-task-list__task-completed" data-testid="oasys-confirmed-tag">
    not started
  </strong>
</li>
```

Reply:
(We need to target the parent because of how the `shouldContainTags` method works, although we could create a separate `shouldContainTag` method and pass in a single tag's `strong` element)